components/org.wso2.carbon.identity.oauth/src/test/java/org/wso2/carbon/identity/oauth2/token/JWTTokenIssuerTest.java

```diff
@@ -766,6 +766,8 @@ public JWTClaimsSet answer(InvocationOnMock invocationOnMock) throws Throwable {
 
         when(mockOAuthServerConfiguration.getOpenIDConnectCustomClaimsCallbackHandler()).
                 thenReturn(claimsCallBackHandler);
+        when(mockOAuthServerConfiguration.getJWTAccessTokenOIDCClaimsHandler()).
+                thenReturn(claimsCallBackHandler);
 
     }
 
```

Comment:
Let's add a comprehensive test class for JWTAccessTokenOIDCClaimsHandler

Reply:
will add tests